UWP fonts

Can't get fonts on UWP, at least FontAwesome and Ionicons. Probably you should follow this advise http://stackoverflow.com/a/35476577/2987062  and check "Generate library layout" option in the Build configuration

Maybe I go wrong, but I guess now you have to include assets folder into nuget fonts packages somehow. I have 

> C:\Program Files (x86)\MSBuild\Microsoft\.NetNative\Microsoft.NetNative.targets(396,7): error MSB3030: Could not copy the file "...\packages\Xam.Plugin.Iconize.FontAwesome\1.0.9\lib\UAP10\Plugin.Iconize.FontAwesome.UWP\Assets\Fonts\iconize-fontawesome.ttf" because it was not found.
> C:\Program Files (x86)\MSBuild\Microsoft\.NetNative\Microsoft.NetNative.targets(396,7): error MSB3030: Could not copy the file "...\packages\Xam.Plugin.Iconize.Ionicons\1.0.9\lib\UAP10\Plugin.Iconize.Ionicons.UWP\Assets\Fonts\iconize-ionicons.ttf" because it was not found.

Just try your own samples in another solution and reference all dependencies through nuget

--- a/Iconize/NuGet/Xam.Plugin.Iconize.EntypoPlus.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.EntypoPlus.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\Entypo+\Plugin.Iconize.EntypoPlus.UWP\bin\Release\Plugin.Iconize.EntypoPlus.UWP.pdb" target="lib\UAP10\Plugin.Iconize.EntypoPlus.UWP.pdb" />
     <file src="Fonts\Entypo+\Plugin.Iconize.EntypoPlus.UWP\bin\Release\Plugin.Iconize.EntypoPlus.UWP.pri" target="lib\UAP10\Plugin.Iconize.EntypoPlus.UWP.pri" />
     <file src="Fonts\Entypo+\Plugin.Iconize.EntypoPlus.UWP\bin\Release\Plugin.Iconize.EntypoPlus.UWP.xml" target="lib\UAP10\Plugin.Iconize.EntypoPlus.UWP.xml" />
-    
+    <file src="Fonts\Entypo+\iconize-entypoplus.ttf" target="lib\UAP10\Plugin.Iconize.EntypoPlus.UWP\Assets\Fonts\iconize-entypoplus.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\Entypo+\Plugin.Iconize.EntypoPlus.Droid\bin\Release\Plugin.Iconize.EntypoPlus.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.EntypoPlus.Droid.dll" />
     <file src="Fonts\Entypo+\Plugin.Iconize.EntypoPlus.Droid\bin\Release\Plugin.Iconize.EntypoPlus.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.EntypoPlus.Droid.pdb" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.FontAwesome.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.FontAwesome.nuspec
@@ -30,6 +30,7 @@
     <file src="Fonts\FontAwesome\Plugin.Iconize.FontAwesome.UWP\bin\Release\Plugin.Iconize.FontAwesome.UWP.pdb" target="lib\UAP10\Plugin.Iconize.FontAwesome.UWP.pdb" />
     <file src="Fonts\FontAwesome\Plugin.Iconize.FontAwesome.UWP\bin\Release\Plugin.Iconize.FontAwesome.UWP.pri" target="lib\UAP10\Plugin.Iconize.FontAwesome.UWP.pri" />
     <file src="Fonts\FontAwesome\Plugin.Iconize.FontAwesome.UWP\bin\Release\Plugin.Iconize.FontAwesome.UWP.xml" target="lib\UAP10\Plugin.Iconize.FontAwesome.UWP.xml" />
+    <file src="Fonts\FontAwesome\iconize-fontawesome.ttf" target="lib\UAP10\Plugin.Iconize.FontAwesome.UWP\Assets\Fonts\iconize-fontawesome.ttf" />
     
     <!--Xamarin.Android-->
     <file src="Fonts\FontAwesome\Plugin.Iconize.FontAwesome.Droid\bin\Release\Plugin.Iconize.FontAwesome.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.FontAwesome.Droid.dll" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.Ionicons.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.Ionicons.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\Ionicons\Plugin.Iconize.Ionicons.UWP\bin\Release\Plugin.Iconize.Ionicons.UWP.pdb" target="lib\UAP10\Plugin.Iconize.Ionicons.UWP.pdb" />
     <file src="Fonts\Ionicons\Plugin.Iconize.Ionicons.UWP\bin\Release\Plugin.Iconize.Ionicons.UWP.pri" target="lib\UAP10\Plugin.Iconize.Ionicons.UWP.pri" />
     <file src="Fonts\Ionicons\Plugin.Iconize.Ionicons.UWP\bin\Release\Plugin.Iconize.Ionicons.UWP.xml" target="lib\UAP10\Plugin.Iconize.Ionicons.UWP.xml" />
-    
+    <file src="Fonts\Ionicons\iconize-ionicons.ttf" target="lib\UAP10\Plugin.Iconize.Ionicons.UWP\Assets\Fonts\iconize-ionicons.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\Ionicons\Plugin.Iconize.Ionicons.Droid\bin\Release\Plugin.Iconize.Ionicons.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.Ionicons.Droid.dll" />
     <file src="Fonts\Ionicons\Plugin.Iconize.Ionicons.Droid\bin\Release\Plugin.Iconize.Ionicons.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.Ionicons.Droid.pdb" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.Material.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.Material.nuspec
@@ -30,6 +30,7 @@
     <file src="Fonts\Material\Plugin.Iconize.Material.UWP\bin\Release\Plugin.Iconize.Material.UWP.pdb" target="lib\UAP10\Plugin.Iconize.Material.UWP.pdb" />
     <file src="Fonts\Material\Plugin.Iconize.Material.UWP\bin\Release\Plugin.Iconize.Material.UWP.pri" target="lib\UAP10\Plugin.Iconize.Material.UWP.pri" />
     <file src="Fonts\Material\Plugin.Iconize.Material.UWP\bin\Release\Plugin.Iconize.Material.UWP.xml" target="lib\UAP10\Plugin.Iconize.Material.UWP.xml" />
+    <file src="Fonts\Material\iconize-material.ttf" target="lib\UAP10\Plugin.Iconize.Material.UWP\Assets\Fonts\iconize-material.ttf" />
 
     <!--Xamarin.Android-->
     <file src="Fonts\Material\Plugin.Iconize.Material.Droid\bin\Release\Plugin.Iconize.Material.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.Material.Droid.dll" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.Meteocons.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.Meteocons.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\Meteocons\Plugin.Iconize.Meteocons.UWP\bin\Release\Plugin.Iconize.Meteocons.UWP.pdb" target="lib\UAP10\Plugin.Iconize.Meteocons.UWP.pdb" />
     <file src="Fonts\Meteocons\Plugin.Iconize.Meteocons.UWP\bin\Release\Plugin.Iconize.Meteocons.UWP.pri" target="lib\UAP10\Plugin.Iconize.Meteocons.UWP.pri" />
     <file src="Fonts\Meteocons\Plugin.Iconize.Meteocons.UWP\bin\Release\Plugin.Iconize.Meteocons.UWP.xml" target="lib\UAP10\Plugin.Iconize.Meteocons.UWP.xml" />
-    
+    <file src="Fonts\Meteocons\iconize-meteocons.ttf" target="lib\UAP10\Plugin.Iconize.Meteocons.UWP\Assets\Fonts\iconize-meteocons.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\Meteocons\Plugin.Iconize.Meteocons.Droid\bin\Release\Plugin.Iconize.Meteocons.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.Meteocons.Droid.dll" />
     <file src="Fonts\Meteocons\Plugin.Iconize.Meteocons.Droid\bin\Release\Plugin.Iconize.Meteocons.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.Meteocons.Droid.pdb" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.SimpleLineIcons.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.SimpleLineIcons.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\SimpleLineIcons\Plugin.Iconize.SimpleLineIcons.UWP\bin\Release\Plugin.Iconize.SimpleLineIcons.UWP.pdb" target="lib\UAP10\Plugin.Iconize.SimpleLineIcons.UWP.pdb" />
     <file src="Fonts\SimpleLineIcons\Plugin.Iconize.SimpleLineIcons.UWP\bin\Release\Plugin.Iconize.SimpleLineIcons.UWP.pri" target="lib\UAP10\Plugin.Iconize.SimpleLineIcons.UWP.pri" />
     <file src="Fonts\SimpleLineIcons\Plugin.Iconize.SimpleLineIcons.UWP\bin\Release\Plugin.Iconize.SimpleLineIcons.UWP.xml" target="lib\UAP10\Plugin.Iconize.SimpleLineIcons.UWP.xml" />
-    
+    <file src="Fonts\SimpleLineIcons\iconize-simplelineicons.ttf" target="lib\UAP10\Plugin.Iconize.SimpleLineIcons.UWP\Assets\Fonts\iconize-simplelineicons.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\SimpleLineIcons\Plugin.Iconize.SimpleLineIcons.Droid\bin\Release\Plugin.Iconize.SimpleLineIcons.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.SimpleLineIcons.Droid.dll" />
     <file src="Fonts\SimpleLineIcons\Plugin.Iconize.SimpleLineIcons.Droid\bin\Release\Plugin.Iconize.SimpleLineIcons.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.SimpleLineIcons.Droid.pdb" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.Typicons.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.Typicons.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\Typicons\Plugin.Iconize.Typicons.UWP\bin\Release\Plugin.Iconize.Typicons.UWP.pdb" target="lib\UAP10\Plugin.Iconize.Typicons.UWP.pdb" />
     <file src="Fonts\Typicons\Plugin.Iconize.Typicons.UWP\bin\Release\Plugin.Iconize.Typicons.UWP.pri" target="lib\UAP10\Plugin.Iconize.Typicons.UWP.pri" />
     <file src="Fonts\Typicons\Plugin.Iconize.Typicons.UWP\bin\Release\Plugin.Iconize.Typicons.UWP.xml" target="lib\UAP10\Plugin.Iconize.Typicons.UWP.xml" />
-    
+    <file src="Fonts\Typicons\iconize-typicons.ttf" target="lib\UAP10\Plugin.Iconize.Typicons.UWP\Assets\Fonts\iconize-typicons.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\Typicons\Plugin.Iconize.Typicons.Droid\bin\Release\Plugin.Iconize.Typicons.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.Typicons.Droid.dll" />
     <file src="Fonts\Typicons\Plugin.Iconize.Typicons.Droid\bin\Release\Plugin.Iconize.Typicons.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.Typicons.Droid.pdb" />

--- a/Iconize/NuGet/Xam.Plugin.Iconize.WeatherIcons.nuspec
+++ b/Iconize/NuGet/Xam.Plugin.Iconize.WeatherIcons.nuspec
@@ -30,7 +30,8 @@
     <file src="Fonts\WeatherIcons\Plugin.Iconize.WeatherIcons.UWP\bin\Release\Plugin.Iconize.WeatherIcons.UWP.pdb" target="lib\UAP10\Plugin.Iconize.WeatherIcons.UWP.pdb" />
     <file src="Fonts\WeatherIcons\Plugin.Iconize.WeatherIcons.UWP\bin\Release\Plugin.Iconize.WeatherIcons.UWP.pri" target="lib\UAP10\Plugin.Iconize.WeatherIcons.UWP.pri" />
     <file src="Fonts\WeatherIcons\Plugin.Iconize.WeatherIcons.UWP\bin\Release\Plugin.Iconize.WeatherIcons.UWP.xml" target="lib\UAP10\Plugin.Iconize.WeatherIcons.UWP.xml" />
-    
+    <file src="Fonts\WeatherIcons\iconize-weathericons.ttf" target="lib\UAP10\Plugin.Iconize.WeatherIcons.UWP\Assets\Fonts\iconize-weathericons.ttf" />
+
     <!--Xamarin.Android-->
     <file src="Fonts\WeatherIcons\Plugin.Iconize.WeatherIcons.Droid\bin\Release\Plugin.Iconize.WeatherIcons.Droid.dll" target="lib\MonoAndroid10\Plugin.Iconize.WeatherIcons.Droid.dll" />
     <file src="Fonts\WeatherIcons\Plugin.Iconize.WeatherIcons.Droid\bin\Release\Plugin.Iconize.WeatherIcons.Droid.pdb" target="lib\MonoAndroid10\Plugin.Iconize.WeatherIcons.Droid.pdb" />

